_examples/mouse.go

```diff
@@ -72,6 +72,14 @@ func keybindings(g *gocui.Gui) error {
 	if err := g.SetKeybinding("msg", gocui.MouseLeft, gocui.ModNone, delMsg); err != nil {
 		return err
 	}
+	// These keys are global, but they work only when you click them while hovering over View.
+	// ModAlt is actually `CTRL` key, not `ALT` key. Tcell generates it like that, not sure why.
+	if err := g.SetKeybinding("", gocui.MouseRight, gocui.ModAlt, delMsg); err != nil {
```

Comment:
How should this work?  
When i test this locally i can just middle mouse click something without having ALT nor CTRL pressed.

Reply:
To be honest, I'm not sure if this should stay there. I added it for testing, but now as I'm looking into Windows 7 problem, I can see that this key combination is registered only on Windows Terminal (win10) connected to Linux machine.

I debugged the keys there, and `Alt` modifier with mouse was not reported with `tcell`. Only `Ctrl` was reported and surprisingly it was reported as `Alt` modifier.     
However, I can totally see this not working on different platforms, or acting differently. 

I can remove it from the example, and keep it as simple right click. Without the modifier it should work without problem. But I do not know if it's not used by other projects somehow.

Reply:
> However, I can totally see this not working on different platforms, or acting differently.

In that case i think we should not add this to the mouse example.  
Maybe add an extra comment to `SetKeybinding` about the mouse modifiers

Reply:
changed in f84d3fd